README.md

```diff
@@ -38,15 +71,17 @@ and we can begin discussion on implementation & planning.
 ## Links
 
 * [Quick-start](docs/quick-start)
+* [Developing quick-start](docs/developing.md)
 * [Resource types](docs/supported-types/README.md)
   * [ElasticsearchCluster](docs/supported-types/elasticsearch-cluster.md)
 
 
 ## E2E Testing
 
-Navigator has an end-to-end test suite which verifies that Navigator can be installed [as documented in the quick start guide](docs/quick-start).
-The tests are run on a Minikube cluster.
-Run the tests using the following sequence of commands:
+Navigator has an end-to-end test suite which verifies that Navigator can be
+installed [as documented in the quick start guide](docs/quick-start).  The
+tests are run on a Minikube cluster.  Run the tests using the following
+sequence of commands:
 
 ```
 minikube start
```

Comment:
These instructions are now out of date and should be removed / updated / moved to the developing section.